FormGenerator.js

```diff
@@ -1,4 +1,4 @@
-// HTML Form generation tool
+// HTML Form Generation Tool
```

Comment:
Looking great! Suggest trying to compress the various input types into a function, to reduce code repetition but the level of attributes you provide is great. Also, on the errors, consider using an error object as mentioned on FSO, it's really simple `throw new Error(message)`. Keep it up!

Reply:
I updated the attribute addition into a single function to reduce code repetition. Error reporting functionality may be implemented better to come, but am going to leave as simple console for now.